docker-compose.yaml

```diff
@@ -22,13 +27,53 @@ services:
     build: .
     volumes:
      - .:/dockermount
+    depends_on:
+      - usaspending-db
     command: python3 -u manage.py shell
     environment:
       DJANGO_DEBUG: ${DJANGO_DEBUG}
       DATABASE_URL: postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/data_store_api
       ES_HOSTNAME: ${ES_HOST}:${ES_PORT}
       DATA_BROKER_DATABASE_URL: postgresql://${BROKER_USER}:${BROKER_PASSWORD}@${BROKER_HOST}:${BROKER_PORT}/data_broker
 
+  usaspending-test:
+    build: .
+    volumes:
+     - .:/dockermount
+    depends_on:
+      - usaspending-db
+      - usaspending-es
+    command: pytest
+    environment:
+      DATABASE_URL: postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/data_store_api
+      ES_HOSTNAME: ${ES_HOST}:${ES_PORT}
+      DATA_BROKER_DATABASE_URL: postgresql://${BROKER_USER}:${BROKER_PASSWORD}@${BROKER_HOST}:${BROKER_PORT}/data_broker
+
+  usaspending-ci:
+    build: .
+    volumes:
+     - .:/dockermount
+    depends_on:
+      - usaspending-db
+      - usaspending-es
```

Comment:
Same potential dependency problem ... 
Since this whole ci check is a series of shell commands, you could just add the waits up front.